tests-ng/test_ssh.py

```diff
@@ -54,6 +54,7 @@
 @pytest.mark.root(reason="Calling sshd -T requires root")
 @pytest.mark.feature("ssh")
 @pytest.mark.parametrize("sshd_config_item", required_sshd_config)
+@pytest.mark.skipif(feature="_fips", reason="FIPS uses different values")
```

Comment:
We're currently not using `@pytest.mark.skipif` and it is not part of the [common filtering patterns](https://github.com/gardenlinux/gardenlinux/blob/0e5025ced68a6712e4a86234c07b583ab14bc045/tests-ng/DEVELOPER.md#common-filtering-patterns) currently. We use `@pytest.mark.feature("not _fips")` (which is a custom plugin) throughout many tests currently

Reply:
Pytest does not know what a "feature" is, so we needed to extend it like we did in `tests-ng/plugins/features.py`.